src/services/WebViews.ts

```diff
@@ -185,6 +201,42 @@ export class WebViews {
 			this.log.error(err);
 		}
 	}
+
+	private async assignUser(assignUser: any, rows: any) {
+		let scanId= this.scanNode.scanId;
+		let nodes = this.queryNode.Result;
+
+		for (var i = 0; i < rows.length; i++) {
+			var pathId = rows[i];
+			for (let nodeCtr = 0; nodeCtr < nodes.length; nodeCtr++) { 
+				if( pathId == nodes[nodeCtr].Path[0].$.PathId) {
+					const request = {"userAssignment" : assignUser};
+					try {
+						await this.httpClient.patchRequest(`sast/scans/${scanId}/results/${pathId}`, request);
```

Comment:
Confirm that below Tests are performed
1. Assign single vulnerability to a user
2. Repeat for another user for different vulnerability. Refresh results and verify that vulnerabilities are now assigned to respective users.
3. Build assign user to multiple vulnerabilities in one go.
Confirm that these tests are added on the work item.

Reply:
Done